### DISCUSSION TITLE
组件的相关属性，对应的对象在哪看对象信息呀？？？

### DISCUSSION THREAD

**Discussion by YaoTeng666** [zouyaoji/vue-cesium#371] [Q&A]
例如：
![image](https://user-images.githubusercontent.com/32535510/226858819-71ceba1c-c9b1-47cb-9b1f-d388a3baa9be.png)
Array<DynamicOverlayOpts>中的DynamicOverlayOpts这个，在哪看这个对象的相关属性呀

**Comment by zouyaoji** [comment]
抱歉了，文档还不够完善，
如果是ts项目vue-cesium是直接导出了的，可以导入后F12跳转看ts声明。

如果是非ts项目，就要配合源码才看了，

[DynamicOverlayOpts ](https://github.com/zouyaoji/vue-cesium/blob/dev/packages/utils/types.ts#L144)
